addons/binding/org.openhab.binding.systeminfo/ESH-INF/thing/channels.xml

```diff
@@ -170,6 +173,14 @@
 		<config-description-ref uri="systeminfo:channels:highpriority" />
 	</channel-type>
 
+	<channel-type id="usedPercent">
```

Comment:
Why 'also'? The other channels in the improved channel groups aren't advanced either.